### ISSUE TITLE
Accordion - On focus and hover contrast ratio issue

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/aria-practices#2360]
- **URL**: https://www.w3.org/WAI/ARIA/apg/example-index/accordion/accordion.html
- **Accessibility failure**: on hover or focus contrast ratio between text and background is no longer sufficient - 1.4.3: Contrast (Minimum)
- **Expected result**: on hover or focus contrast ratio between text and background is sufficient
- **WCAG reference**: The minimum contrast success criterion (1.4.3) applies to text in the page, including placeholder text and text that is shown when a pointer is hovering over an object or when an object has keyboard focus. If any of these are used in a page, the text needs to provide sufficient contrast.
- **Issue snapshot**: <img width="1792" alt="screenshot highlighting poor contrast ratio between accordion button text and background on focus" src="https://user-images.githubusercontent.com/106394951/170731654-c6bc714b-e076-4e62-a4f7-ad80bb511e89.png">
- **Issue Description**: While navigating the page using the keyboard, once focus (or hover) is moved on buttons that trigger accordions to be expanded, the contrast ratio between text and background is no longer sufficient.

Giacomo Petri

**Comment by mcking65** [comment]
Duplicate of w3c/wai-aria-practices#105

**Comment by jnurthen** [comment]
also a dupe of https://github.com/w3c/aria-practices/issues/2333
